include/cufinufft.h

```diff
@@ -5,13 +5,31 @@
 #include <cstdlib>
 #include <assert.h>
 #include <cuda_runtime.h>
-#include "../finufft/utils.h"
-#include "../finufft/spreadinterp.h"
-#include "../finufft/finufft.h"
-#include "../finufft/common.h"
+#include "../contrib/utils.h"
+#include "../contrib/spreadinterp.h"
+#include "../contrib/common.h"
 
 enum finufft_type {type1,type2,type3};
 
+struct nufft_opts {   // see cufinufft_default_opts() for defaults
```

Comment:
Since these options are specific to cufinufft, should it perhaps be called `cufinufft_opts`? Also, doesn't this need a `typedef`?

Reply:
See 4f89eeb.